share/smack/top.py

```diff
@@ -846,8 +859,8 @@ def transform_out(args, old):
 
 def verification_result(verifier_output, verifier):
     if re.search(
-        r'[1-9]\d* time out|Z3 ran out of resources|timed out|ERRORS_TIMEOUT',
-            verifier_output):
+            r'[1-9]\d* time out|Z3 ran out of resources|timed out|ERRORS_TIMEOUT',
```

Comment:
Why did you introduce this additional indentation here? Was that required for formatting to go through?